e2e_tests/Makefile

```diff
@@ -20,7 +20,7 @@ check:
 		-s ./tests/fixtures/pytorch_lightning_amp/mnist.py \
 		-s ./tests/fixtures/pytorch_lightning_amp/data.py
 	black . --check
-	flake8
+	python -m flake8
```

Comment:
As you pointed out earlier, this is odd. Maybe worth a comment since it represents a real, engineering decision? Up to you.